tests/FsPickler.Tests/SerializerTests.fs

```diff
@@ -1177,3 +1177,26 @@ type SerializationTests (fixture : ISerializerFixture) =
             raise <| new AssertionException(msg)
         else
             printfn "Failed Serializations: %d out of %d." failedResults results.Length
+
+     // Large array tests output so much data they won't fit into a .NET byte array, we have to serialize to a FileStream.
+    [<Test; Category("Stress tests")>]
```

Comment:
I'm trying to understand the motivation of this PR. Is this test failing or not performing well in the current implementation?

Reply:
Need to up the size, but I've got two tests one with 2GB of int64s and one with 5GB of a 20byte struct. Both fail on master.